src/symbolic_dimensions.jl

```diff
@@ -1,26 +1,32 @@
 import .Units: UNIT_SYMBOLS, UNIT_MAPPING, UNIT_VALUES
 import .Constants: CONSTANT_SYMBOLS, CONSTANT_MAPPING, CONSTANT_VALUES
 
+
 const SYMBOL_CONFLICTS = intersect(UNIT_SYMBOLS, CONSTANT_SYMBOLS)
 
+disambiguate_symbol(s) = s in SYMBOL_CONFLICTS ? Symbol(string(s) * "_constant") : s
+
 const INDEX_TYPE = UInt8
 # Prefer units over constants:
 # For example, this means we can't have a symbolic Planck's constant,
 # as it is just "hours" (h), which is more common.
 const ALL_SYMBOLS = (
     UNIT_SYMBOLS...,
-    setdiff(CONSTANT_SYMBOLS, SYMBOL_CONFLICTS)...
-)
-const ALL_VALUES = vcat(
-    UNIT_VALUES...,
-    (
-        v
-        for (k, v) in zip(CONSTANT_SYMBOLS, CONSTANT_VALUES)
-        if k ∉ SYMBOL_CONFLICTS
-    )...
+    disambiguate_symbol.(CONSTANT_SYMBOLS)...
 )
+const ALL_VALUES = (UNIT_VALUES..., CONSTANT_VALUES...)
```

Comment:
BTW is there a particular reason for using a tuple for `ALL_SYMBOL` instead of an array? And is `ALL_VALUES` actually needed? Every entry should be available as `getproperty(Units, symbol::Symbol)` and `getproperty(Constants, symbol::Symbol)`?

Reply:
I admit I don't know where the benefits of tuple-ifying constants end. Where would you draw the line? I'm assuming small tuples help with extra allocations. Are there downsides to using large tuples, if they are declared constants?

Reply:
> Every entry should be available as `getproperty(Units, symbol::Symbol)` and `getproperty(Constants, symbol::Symbol)`?

Nice!! I'll do that.

Reply:
Wait, nevermind re: this^. The combined tuples here simplify things as I don't have to check whether the symbol is in Units or Constants, I just look up the index.